### ISSUE TITLE
Help : How to catch events

### ISSUE THREAD

**Issue by Shigehiro** [akamai/nomcc#2]
It would be helpful if there is a sample code to catch events in the README.

**Comment by nombwelling** [comment]
Agreed.  I just opened https://github.com/akamai/nomcc/pull/3 to add such an example, include documentation of the dispatch protocol, and slightly simplify the interface.

Until that's merged, the example in the PR will mostly work, except that the dispatch parameter to `nomcc.connect()` would need be replaced with a call to `session.set_dispatch(print_event)`.

**Comment by Shigehiro** [comment]
Thank you for adding the example.
